### PR TITLE
[IMP] inventory: update automated inventory valuation

### PR DESCRIPTION
**Pull request by brse-odoo**
Issue #2842 

Addressed concerns listed in Issue 2842.

- Fixed 100th character line breaks
- Fixed indentations
- Fixed typos and improved grammar
- Added abbreviation tags to POs

### CODE REVIEW

**Comment by StraubCreative** [comment]
ToDo: need to take a look at the examples inside `documentation/static/js/coa-valuation-continental.js` per items outlined outlined in [this issue comment](https://github.com/odoo/documentation/issues/2842#issuecomment-1373413982).

**Comment by StraubCreative** [comment]
Hi @jcs-odoo 

Requesting your review for a proposed rewrite + restructure for this entire doc.

I pushed a proof-of-concept on [a27d515](https://github.com/odoo/documentation/pull/3242/commits/a27d515fa67858d6fb89113b62528c3910408b72) without squash so you can compare the diffs.

The biggest changes here, are that we expanded on the written content to more clearly define terms and then went into specific configuration steps for inventory valuation. We did away with the tables too (which included the JS files the tables relied upon, since they were no longer needed) given the feedback we received from users and local PE that the tables were too confusing or simply not needed.

I _think_ these changes fully address the issues raised #2842 by more clearly laying out valuation options for logistics and accounting teams, and removing demo/play data that otherwise might better be covered in eLearning as a Tutorial or a Use Case.

Since you're on both @odoo/user-docs-structure-review and @odoo/accounting-doc-review teams, we'd love to hear your feedback on this 🙂 

TIA/KR 

cc: @brse-odoo @samueljlieber @AntoineVDV

**Comment by AntoineVDV** [comment]
Considering how painful it is to support page-specific React JS and how negative the PR's line diff is, I see this decision as a no-brainer :+1:

File: content/applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst [hunk 1]
```diff
@@ -1,424 +1,133 @@
-:code-column:
-:custom-css: accounting.css
-:custom-js: coa-valuation-continental.js,coa-valuation-anglo-saxon.js,misc.js
-
-=================================
+---------------------------------
```
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst:1]: Keep using `============` for H1.

**Review by AntoineVDV — COMMENTED** [review]
@robodoo delegate+

**Comment by samueljlieber** [comment]
1a7f970 to fix H1 formatting :)

**Comment by brse-odoo** [comment]
robodo r+

**Review by StraubCreative — APPROVED** [review]

**Review by jcs-odoo — COMMENTED** [review]
Hi @StraubCreative @brse-odoo 

Sorry I answer this message so late.

I trust you picked a good solution in describing each point more. From what I read, it's certainly clearer :)

Still, I think the tables and exercises were useful in understanding the valuation. Probably not _these_ tables, especially these dynamic ones. And I guess you could add them in a later PR.

You could provide some examples in "types of accounting", split into several sub-headings...
(maybe in an "example admonition", or dedicated section), with group-tabs in it (one for continental, one for Anglo-saxon) which makes the comparison of entries easier to understand, and skippable if needed.

I don't know if it's clear... Let me give you a quick draft ^^ 

There, it would roughly like this
image

And here is the code

```
Examples
~~~~~~~~

.. tabs::

   .. group-tab:: Anglo-Saxon

      Entries blablabla Lorem Ipsum you are an anglo-saxon wizard, Harry!

      .. list-table::
         :header-rows: 1
         :stub-columns: 1

         * -
           - Debit
           - Credit
         * - Assets: Inventory
           - 50
           -
         * - Assets: Deferred Tax Assets
           - 4.68
           -
         * - Liabilities: Accounts Payable
           -
           - 54.68

      End of the year blablabla

      .. list-table::
         :header-rows: 1
         :stub-columns: 1

         * -
           - Debit
           - Credit
         * - Assets: Inventory (closing value)
           - X
           -
         * - Expenses: Cost of Good Sold
           - X
           -
         * - Expenses: Purchased Goods
           -
           - X
         * - Assets: Inventory (starting value)
           -
           - X

   .. group-tab:: Continental

      Entries blablabla Lorem Ipsum you are a continental wizard, Harry!

      .. list-table::
         :header-rows: 1
         :stub-columns: 1

         * -
           - Debit
           - Credit
         * - Assets: Inventory
           - 50
           -
         * - Assets: Deferred Tax Assets
           - 4.68
           -
         * - Liabilities: Accounts Payable
           -
           - 54.68

      End of the year blablabla

      .. list-table::
         :header-rows: 1
         :stub-columns: 1

         * -
           - Debit
           - Credit
         * - Assets: Inventory
           - X
           -
         * - Expenses: Inventory Variations
           -
           - X
```

Same thing for the exercises at the end: you could easily have several options displayed cleanly with a table.

It's only a suggestion, but I think these journal help in learning more about inventory valuation.

Also, @brse-odoo , it seems your r+ didn't go through.

**Comment by brse-odoo** [comment]
Hi @jcs-odoo, thank you for looking this over and providing feedback! I have cleaned up the suggestions you left on lines 36 and 70 and pushed up the changes.

Also, @StraubCreative and I will be addressing the suggestions for re-adding the accounting tables in a separate PR once we come up with examples together -- thank you for providing a template for us!

**Comment by brse-odoo** [comment]
@robodoo r+

**Comment by brse-odoo** [comment]
@robodoo r+

**Comment by brse-odoo** [comment]
@robodoo r+

**Pull request closed without merging**